platform/vs/docker-sonic-vs/database_config.json

```diff
@@ -58,9 +58,24 @@
             "separator": "|",
             "instance" : "redis"
         },
-        "CHASSIS_DB" : {
+        "GB_ASIC_DB" : {
```

Comment:
can you move after chassis db for backward compatibility?

Reply:
@lguohan I'm currently in the process of trying to build with latest upstream and my changes here locally to investigate. One problem here is that is swss-swss-common there is a and config and header file that defines these tables. I did not see an update to that project for this chassis db, so that is why I rearranged things here to be compatible with sonic-swss-common. Perhaps it is a defect that the chassis code did not change this - see https://github.com/Azure/sonic-swss-common/blob/master/common/database_config.json and https://github.com/Azure/sonic-swss-common/blob/master/common/schema.h where this is no refernec eto chassis db and we already claimed table 8.

Reply:
Note there is probably some room for refactoring here so we only have one such config file, I'm not sure what led to this database schema needing to be defined in multiple places, might be good instead of #defines to be able to query table names at runtime based on the schema.

Reply:
CHASSIS_DB change is the first PR of SONIC chassis support, please check chassis sub group work. Many following PRs will depend on it.

Reply:
@BrynXu  Thanks for the response. So, you should see a merge conflict then based on changes already in sonic-swss-config. We'd prefer if you could use 11 as DB number for CHASSIS_DB, as I have done in my pull request. We are currently working on last of 5 pull requests for adding gearbox phy support to vs builds.

Numbering is current in upstream as follows:

...

#define GB_ASIC_DB      8
#define GB_COUNTERS_DB  9
#define GB_FLEX_COUNTER_DB  10